src/website/views/components/picker-group.pug

```diff
@@ -0,0 +1,252 @@
+        <label for="checkbox3">
+          Checkbox3
+        </label>
+      </div>
+    </div>
+
+h4 Alignment
+p.-text
+  | Button text is centered by default. However, this behavior can change by applying the modifiers
+  | <code>-align--left</code> or <code>-align--right</code>.
+.example.-mb--3
+  .-p--3
+    .m-pickerGroup.-fluid
+        .m-picker
+          input(type="radio", class="a-picker", name='box', id='id-Radio1')
+          label(for='id-Radio1', class=`-align--left`) Radio1
+        .m-picker
+          input(type="radio", class="a-picker", name='box', id='id-Radio2')
+          label(for='id-Radio2', class=`-align--center`) Radio2
+        .m-picker
+          input(type="radio", class="a-picker", name='box', id='id-Radio3')
+          label(for='id-Radio3', class=`-align--right`) Radio3
+  :code(lang='html')
+    <div class="m-pickerGroup -fluid">
+      <div class="m-picker">
+        <input class="a-picker" type="checkbox" id="checkbox1">
+        <label class="-align--left" for="checkbox1">
+          Checkbox1
+        </label>
+      </div>
+      <div class="m-picker">
+        <input class="a-picker" type="checkbox" id="checkbox2">
+        <label class="-align--center" for="checkbox2">
+          Checkbox2
+        </label>
+      </div>
+      <div class="m-picker">
+        <input class="a-picker" type="checkbox" id="checkbox3">
+        <label class="-align--right" for="checkbox3">
+          Checkbox3
+        </label>
+      </div>
+    </div>
+
+h4 Disable Fluidity
+p.-text
+  | To disable the fluidity of a specific button in a fluid button group, apply the class <code>-notFluid</code> to the button.
+
+.example.-mb--3
+  .-p--3
+    .m-pickerGroup.-fluid
+        .m-picker
+          input(type="radio", class="a-picker", name='box', id='ie-Radio1')
+          label(for='ie-Radio1') Radio1
+        .m-picker.-notFluid
+          input(type="radio", class="a-picker", name='box', id='ie-Radio2')
+          label(for='ie-Radio2') Radio2
+        .m-picker
+          input(type="radio", class="a-picker", name='box', id='ie-Radio3')
+          label(for='ie-Radio3') Radio3
+  :code(lang='html')
+    <div class="m-pickerGroup -fluid">
+      <div class="m-picker">
+        <input class="a-picker" type="checkbox" id="checkbox1">
+        <label for="checkbox1">
+          Checkbox1
+        </label>
+      </div>
+      <div class="m-picker -notFluid">
+        <input class="a-picker" type="checkbox" id="checkbox2">
+        <label for="checkbox2">
+          Checkbox2
+        </label>
+      </div>
+      <div class="m-picker">
+        <input class="a-picker" type="checkbox" id="checkbox3">
+        <label for="checkbox3">
+          Checkbox3
+        </label>
+      </div>
+    </div>
+
+h3 Sizes
+
+p.-text
+  | Button groups supports a full spectrum of sizes: <code>-sm</code>, <code>-md</code>,
+  |  <code>-lg</code>, <code>-xl</code>.
+  | The default size is <code>-md</code>.
+
+.example.-mb--3
```

Comment:
Can you fix this? Examples don't match the code snippet and all sizes are using the same id's so clicking -md, -lg, -xl activates -sm